src/pages/ItemsList/components/FilterBar/FilterBar.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import FilterBarCheckList from './components/FilterBarCheckList';
+import FilterBarColor from './components/FilterBarColor/FilterBarColor';
+import FilterBarSize from './components/FilterBarSize';
+
+function FilterBar(props) {
```

Comment:
```suggestion
function FilterBar() {
```

props로 받을 데이터가 없다면 이 부분은 제거해 주셔도 됩니다!

Reply:
완료